doc/whats_new/v1.4.rst

```diff
@@ -167,6 +167,10 @@ Changelog
 :mod:`sklearn.preprocessing`
 ............................
 
+- |Efficiency| :class:`preprocessing.OrdinalEncoder` avoids calculating
+  `missing_indices` twice to improve efficiency.
```

Comment:
Nit: Given that `missing_indices` is a private attribute, I prefer to write out the term:

```suggestion
  missing indices twice to improve efficiency.
```